packages/cli/src/Logger.ts

```diff
@@ -0,0 +1,79 @@
+import * as winston from 'winston';
+import config = require('../config');
+
+import {
+	ILogger,
+	LogTypes,
+} from 'n8n-workflow';
+
+export class Logger implements ILogger {
+	private logger: winston.Logger;
+	
+	constructor() {
+
+		const logFormat = winston.format.printf(({ message }) => {
+			return message;
+		}) as winston.Logform.Format;
+		
+		this.logger = winston.createLogger({
+			level: config.get('logs.level'),
+			format: logFormat,
+			transports: [
+				new winston.transports.Console(),
+			],
+		});
+		
+		if (config.get('logs.output') === 'file') {
+			const fileLogFormat = winston.format.combine(
+				winston.format.timestamp(),
+				winston.format.json()
+			);
+			this.logger.add(
+				new winston.transports.File({
+					filename: config.get('logs.file.location'),
+					format: fileLogFormat,
+					maxsize: 16 * 1024 * 1024, // 16MB
+					maxFiles: 100, // 100 files of 16MB = 1.6GB
```

Comment:
thanks for addressing this
one more thought - there might be best as configurable values rather than static

Reply:
Thanks again for your contribution @mutdmour 

Just updated with your suggestions!